### ISSUE TITLE
IDEA: Add testimonials to home page

### ISSUE THREAD

**Issue by lpatmo** [codebuddies/codebuddies#914]
From https://medium.com/codebuddies/codebuddies-community-survey-results-really-insightful-commentary-from-yall-8dcb36965d71, we have a couple of survey responses to the question "How did CodeBuddies help you?" that we could highlight as testimonials, e.g.:

> CodeBuddies is a place where people of all kinds meet up to advance their jobs, skills, and knowledge as a team.

> CodeBuddies has helped me by giving me a place to talk about coding without being judged by who I am and what languages and skills I’m learning.

> An active Slack community with a team that genuinely cares. Useful for those learning how to code.

> Very helpful to meet people also looking to improve their skills. Helpful also if you are stuck on a concept or piece of code as it can be posted as snippet for others to help resolve issue.

> I loved seeing all the positivity, especially when I was having a hair-pulling-out day. It’s great to see people helping and giving advice and genuinely being good and nice people to their fellow developers.

> I work remotely with a couple international teams (much smaller US team) and it’s nice to have found some like minded people with regular hours.

=========
We could also explicitly ask some members who've been active to share more testimonials.

**Comment by titchimoto** [comment]
@lpatmo Not sure if this issue is up for grabs? But I would love to have a go at this and make it look nice & pretty.

**Comment by lpatmo** [comment]
It's up for grabs! :)

**Comment by titchimoto** [comment]
Oh, awesome! I’d love to have a go at it, please! Shall I just stick with the above quotes for now, or did you get others, too? I’ll put something together in the next day or two & submit a PR. Thanks!

**Comment by railsstudent** [comment]
Claimed by @titchimoto

**Comment by lpatmo** [comment]
@titchimoto I haven't gotten around to asking folks directly to submit quotes, so the above from the survey are fine. Thank you!

**Comment by titchimoto** [comment]
That's fine! I used all the ones you provided earlier, and I think those work well. 

This is what I have so far in code, is this the kind of thing you were thinking? It's pretty simple but I've tried to keep it nice and clean. 

![testimonials](https://user-images.githubusercontent.com/29615325/42714483-303859b6-86a8-11e8-9a2e-f30e8e5f2e29.png)

Let me know what you think and I'll make a couple more tweaks and submit the PR! :)

**Comment by lpatmo** [comment]
@titchimoto That looks great! Maybe "Don't just take our word for it; these are what some of our members have said anonymously about CodeBuddies"

Are the quotes meant to be wrapped around the text? :)

Will defer to @adachiu on design feedback, in case she has any thoughts.

**Comment by railsstudent** [comment]
How do you do the CSS for the bubble with triangular pointer?  Look awesome.

**Comment by adachiu** [comment]
@lpatmo it's hard to say how it will fit with the revised home page at the moment since we haven't gotten to it yet so I have no comments.

**Comment by titchimoto** [comment]
Fair enough. 

@lpatmo Thanks! I changed the text now to match yours. And I originally tried to get the quotes around the text but it looked weird? Maybe because they're big or something, so I left them as a little icon looking thing instead. I've just fixed a few other little tweaks and spacing issues etc. made the font size smaller, so I'll submit the PR and I'll leave it up to you guys to decide! 

@railsstudent It's actually a little trick using the `:after ` CSS pseudo-class. And then essentially creating a little triangle using CSS and appending it literally after each div. There's a cool blog post I found about it here: http://nicolasgallagher.com/pure-css-speech-bubbles/
